Packs/Anomali_ThreatStream/TestPlaybooks/playbook-Anomali_ThreatStream_Test.yml

```diff
@@ -1,4 +1,4 @@
-id: Anomali_ThreatStream_Test
+id: Anomali ThreatStream Test
```

Comment:
dont we need to update this in `conf.json` as well ?

Reply:
This is a mistake, we decided I won't touch name != id issues, I'll fix that